bin/crew

```diff
@@ -1050,10 +1064,10 @@ def resolve_dependencies
   puts 'The following packages also need to be installed: '
 
   deps = @dependencies
-  begin_packages = []
-  end_packages = []
-  first_packages = %w[curl git pixz shared_mime_info]
-  last_packages = %w[ghc mandb gtk3 gtk4 sommelier]
+  # populate arrays with common elements
+  begin_packages = deps & CREW_FIRST_PACKAGES
+  end_packages = deps & CREW_LAST_PACKAGES
```

Comment:
Why do we need to assign the constants to new variables?  Can't we just use `CREW_FIRST_PACKAGES` and `CREW_LAST_PACKAGES` where needed instead?  BTW, this logic should resolve correctly when installing or building packages.

Reply:
It was working perfectly before you concatenated deps.  There was some refactoring so maybe this is necessary now but I'm still not understanding why it was changed.